design-system/src/theme-provider.visualroute.jsx

```diff
@@ -1,34 +1,42 @@
-import { useTheme, customProperties } from '@commercetools-uikit/design-system';
-import { useLayoutEffect } from 'react';
+import { ThemeProvider, customProperties } from '@commercetools-uikit/design-system';
 import { Suite, Spec } from '../../test/percy';
 
 export const routePath = '/theme-provider';
 
 const DummyComponent = () => (
-    <h1 style={{
+  <h1
+    style={{
       color: customProperties.colorPrimary,
-    }}>
-      Title with default theme <i>colorPrimary</i> design token
-    </h1>
-  );
-
-const DarkThemedComponent = () => {
-  const { changeTheme } = useTheme();
-  useLayoutEffect(() => {
-    changeTheme('dark');
-  }, [changeTheme]);
-
-  return (<DummyComponent />);
-};
+    }}
+  >
+    Title with default theme <i>colorPrimary</i> design token
```

Comment:
The only issue I see is that `default theme` is hardcoded here and rendered in all cases. If you don't feel like changing that here I can take care of that later.

Reply:
Thanks for pointing that out 👍 

Updated in this commit: 9707ed017093bcaf474fe321006de4d3139725b7